admin_manual/file_workflows/access_control.rst

```diff
@@ -8,6 +8,9 @@ rules. If all rules of a group hold true, the group matches the request and
 access is being denied. The rules criteria range from IP address, to user
 groups, collaborative tags and :ref:`some more <available-rules-label>`.
 
+.. note:: Keep in mind that Context Chat is not affected by the File Access Control rules
+    and will index files for which access is denied by a rule.
```

Comment:
```suggestion
.. note:: In case you are using the :ref:`Context Chat App <ai-app-context_chat>`,
    please keep in mind, that it is not affected by the File Access Control rules
    and will respond with indexed information, even when the file is not accessible
    by the user due to access control rules.
```

Didn't try the label, but it should link to https://github.com/nextcloud/documentation/blob/master/admin_manual/ai/app_context_chat.rst?plain=1#L5